#! /bin/sh /usr/share/dpatch/dpatch-run
## rl-header.dpatch by  <doko@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Include stdio.h in readline.h and history.h

@DPATCH@
diff -urNad readline5-5.2~/history.h readline5-5.2/history.h
--- readline5-5.2~/history.h	2003-07-31 14:38:44.000000000 +0200
+++ readline5-5.2/history.h	2006-10-28 14:06:20.000000000 +0200
@@ -32,6 +32,7 @@
 #  include "rlstdc.h"
 #  include "rltypedefs.h"
 #else
+#  include <stdio.h>
 #  include <readline/rlstdc.h>
 #  include <readline/rltypedefs.h>
 #endif
diff -urNad readline5-5.2~/readline.h readline5-5.2/readline.h
--- readline5-5.2~/readline.h	2006-08-16 21:16:59.000000000 +0200
+++ readline5-5.2/readline.h	2006-10-28 14:06:20.000000000 +0200
@@ -33,6 +33,7 @@
 #  include "keymaps.h"
 #  include "tilde.h"
 #else
+#  include <stdio.h>
 #  include <readline/rlstdc.h>
 #  include <readline/rltypedefs.h>
 #  include <readline/keymaps.h>
